chrome/skin/global/msgHdrViewOverlay.css

```diff
@@ -18,14 +18,26 @@
   display: none;
 }
 
+#expandeddkim-verifierBox:not([spf="false"])  [anonid="spf"] {
```

Comment:
Could you try if the following works too? (Same for the other lines)

```suggestion
#expandeddkim-verifierBox[spf="true"]  [anonid="spf"] {
```

Makes more sense to me than the double negation.

Reply:
Thank you :)